epub34/authoring/index.html

```diff
@@ -7602,30 +7531,29 @@ <h5>Spread placement</h5>
 							reflowable content. They only apply when the reading system is creating synthetic
 							spreads.</p>
 
-						<p>Although EPUB creators often indicate to use a spread in certain device orientations, the
-							content itself does not represent true spreads (i.e., two consecutive pages that reading
-							systems must render side-by-side for readability, such as a two-page map). To indicate that
-							two consecutive pages represent a true spread, EPUB creators SHOULD use the
-								<code>rendition:page-spread-left</code> and <code>rendition:page-spread-right</code>
-							properties on the [=EPUB spine | spine=] items for the two adjacent EPUB content documents,
-							and omit the properties on spine items where one-up or two-up presentation is equally
-							acceptable.</p>
-
-						<p>EPUB creators MUST NOT declare more than one <code>rendition:page-spread-*</code> property,
-							and/or their unprefixed equivalents, on any given spine item (e.g., it is valid to specify
-							both "<code>rendition:page-spread-left page-spread-left</code>" in case reading systems only
+						<p>Although it is common practice to specify to use a spread in certain device orientations, the
+							content itself does not represent a true spread &#8212; two consecutive pages that have to
```

Comment:
```suggestion
						<p>Although it is common practice to specify the use a spread in certain device orientations, the
							content itself does not represent a true spread &#8212; two consecutive pages that have to
```

Reply:
Needs an "of" to be grammatically correct
```suggestion
						<p>Although it is common practice to specify the use of a spread in certain device orientations, the
							content itself does not represent a true spread &#8212; two consecutive pages that have to
```